preupg/cli.py

```diff
@@ -54,126 +54,166 @@ def __init__(self, args=None):
 
     def add_args(self):
         self.parser.add_option(
-            "--skip-common",
+            "-h", "--help",
+            action="help",
+            help="Show help message and exit."
+        )
+        self.parser.add_option(
+            "-S", "--skip-common",
             action="store_true",
-            help="Skip generation of common files"
+            help="Skip generating files containing information about the"
+                 " system that are used by modules. For assessing the system"
+                 " these files are needed but they can be reused from the"
+                 " previous runs of Preupgrade Assistant."
         )
```

Comment:
The text could point out that the files are safely reusable only in case that system is in "same" state since the files were generated. That means: are installed same packages, previously unchanged configuration files are still unchanged, ... Otherwise it is recommended to create common files each run.

Note:
Probably this option would be deprecated in future, as we split logic to modules itself. Option is usable mainly for developers of the modules and preupgrade-assistant, so I think it would be mabye replaced by new directive in config file and available only under devel-mode section.